src/middlewares/analyticsMiddleware/index.js

```diff
@@ -0,0 +1,19 @@
+import { newWalletEvents } from './newWalletEvents'
+import { sendTransactionEvents } from './sendTransactionEvents'
+import { userParams } from './userParams'
+
+export const analyticsMiddleware = store => next => (action) => {
```

Comment:
would be better if we don't duplicate `middleware` word for any middleware.
Obvious that they are middleware, because they are placed in `middlewares` dir